### PR TITLE
Hide Queries fields

### LINKED ISSUE #321: Hide fields of Query objects under getters
It'll allow us to be more flexible in future without breaking the API

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #321 

@nikitin-da PTAL

### CODE REVIEW

**Comment by nikitin-da** [comment]
Emission hasn't caught, may be we need to increase waiting time?

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/DeleteQuery.java [hunk 1]
```diff
@@ -11,40 +11,63 @@
...
-     * Optional filter declaring which rows to return
+     * Gets {@code WHERE} clause.
      * <p/>
-     * Formatted as an SQL WHERE clause (excluding the WHERE itself).
+     * Optional filter declaring which rows to return.
      * <p/>
-     * Passing null will return all rows for the given table
+     * Formatted as an SQL {@code WHERE} clause (excluding the {@code WHERE} itself).
+     * <p/>
+     * If it's {@code null} — Query will retrieve all rows for the given table.
```
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/DeleteQuery.java]: Do you mean `Query will delete all rows`?

**Comment by artem-zinnatullin** [comment]
Damn, I'll create a bug for this, I face this on my Core i7 Desktop from
time to time, I don't think that it's the problem of performance, I'll
check the code of TestObserver, maybe I made a bug in notifications
implementation.

ср, 13 мая 2015, 13:48, Dmitrii Nikitin <notifications@github.com>:

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/DeleteQuery.java (hunk 1 above)
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/DeleteQuery.java]: Yep, my bad, I'll fix this

ср, 13 мая 2015, 13:50, Dmitrii Nikitin <notifications@github.com>:

Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/query/DeleteQuery.java]: Fixed!

**Comment by artem-zinnatullin** [comment]
Rebased and squashed into one commit. @nikitin-da if notifyAboutChangesConcurrently will fail again and it'll be the only failed test — merge this without worries, I'll fix it later.

**Comment by nikitin-da** [comment]
Tests not found

**Comment by nikitin-da** [comment]
retest this please

**Comment by artem-zinnatullin** [comment]
retest this please

**Pull request merged**